### PR TITLE
Fix about & changelog logic

### PR DESCRIPTION
**Pull request by z11h**
hmm, for some reason this PR still has the other PR history...no worries it will fix itself :)

Ok, so for what this PR does:
1. Un-Hard Codes the changelog (I will have it fetch it from an external source later :smile: )
   - This makes it much easier to update the changelog since the page does not have it hardcoded 

TODO/Ideas for future:
- Have the changelog pulled from an external source (CHANGELOG.md? external web page?)
- Have a checkbox that shows the changelog and hides it (very easy to implement with ng-if). Coming in next PR

### CODE REVIEW

**Comment by xaiki** [comment]
i think having it pulled from CHANGELOG.md is much better, and shouldn't be too hard to do

**Comment by z11h** [comment]
@xaiki its been implemented 😄

**Comment by vankasteelj** [comment]
seems good to me :+1:

File: src/client/about/about.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
             Butter is the result of many developers and designers putting a bunch of APIs together <br />
             to make the experience of watching torrent movies as simple as possible.<br />
             We are an open source project. We are from all over the world. We love our movies. <br />
-            And boy, do we love Popcorn with Butter.
+            And boy, do we love our Popcorn with Butter.
```
Comment (z11h) [inline src/client/about/about.html:13]: @vankasteelj wait is this breaking anything?

Comment (vankasteelj) [inline src/client/about/about.html:13]: it's already broken in this case, I think.

**Comment by z11h** [comment]
Are we good with merging this?

**Pull request merged**